sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -724,6 +724,8 @@ private void ValidateMessageHandler()
         /// called while the processor is not running, no action is taken. This method
         /// will not close the underlying receivers, but will cause the receivers to stop
         /// receiving. To close the underlying receivers, <see cref="CloseAsync"/> should be called.
+        /// Any in-flight message handlers will be awaited, and this method will not return
```

Comment:
Would it be worth explicitly calling out that no messages will be processed after this method as returned unless `StartAsync` is called again?